tests/06-sim-actions.tests

```diff
@@ -12,7 +12,7 @@ test type: bpf-sim
 06-sim-actions	all		write		1		0x856B008	N	N	N	N	ERRNO(1)
 06-sim-actions	all		close		4		N		N	N	N	N	TRAP
 06-sim-actions	all		openat		0		0x856B008	4	N	N	N	TRACE(1234)
-06-sim-actions	all		fstat		N		N		N	N	N	N	KILL_PROCESS
+06-sim-actions	all,-loongarch64		fstat		N		N		N	N	N	N	KILL_PROCESS
```

Comment:
It looks like every ABI, including loongarch64, supports `fstatfs`, yes?  If so, let's change the test to use the `fstatfs` syscall so we can avoid the loongarch64 special case.

Reply:
loongach64 use the default syscall set, it does not define `__ARCH_WANT_NEW_STAT` in the `asm/unistd.h` file, so there is no `fstatfs` and `newfstatat`, instead, the `statx` system call is used.

More information here:
https://patches.linaro.org/project/netdev/patch/20180716161103.16239-3-arnd@arndb.de/
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/arch/loongarch/include/uapi/asm/unistd.h?h=v6.0.1
https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/tree/arch/loongarch/include/uapi/asm/unistd.h

Reply:
If loongach64 does not support `fstatfs` then there is a problem in the "src/syscalls.csv" changes as it looks like you are defining `fstatfs` as syscall `44` for loongach64.

https://github.com/seccomp/libseccomp/blob/cafd0d5cfb5ceefc4dbad7ce19961ae09451e0ca/src/syscalls.csv#L97

Reply:
Sorry, I was wrong, there should be no `79` and `80` syscall numbers.
These system calls are removed: `fstatat` `newfstatat` `fstatat64` `fstat` `fstat64`.
`fstatfs` still exists, it's number `44`. 
So the patch doesn't need to be modified, it's right.

Reply:
> Forgive me if I've missed something. It appears that this thread has verified that all archs support `fstatfs`, so @pcmoore's request makes sense. We should be able to change this test's line to
> 
> ```
> 06-sim-actions	all		fstatfs		N		N		N	N	N	N	KILL_PROCESS
> ```

Ok, but I think we should also change the code in 06-sim-actions.c:
```
-       rc = seccomp_rule_add(ctx, SCMP_ACT_KILL_PROCESS, SCMP_SYS(fstat), 0);
+       rc = seccomp_rule_add(ctx, SCMP_ACT_KILL_PROCESS, SCMP_SYS(fstatfs), 0);
```